app/helpers/helper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Helpers;
+
+class Helper
+{
+    public static function index_number($page, $per_page, $index)
+    {
+        $page = $page - 1;
+        if($page < 0) { $page = 0; }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 11</br>Variable "per_page" is not in valid camel caps format